pose_format/utils/holistic.py

```diff
@@ -52,11 +54,14 @@ def process_holistic(frames: list, fps: float, w: int, h: int, kinect=None, prog
         results = holistic.process(frame)
 
         body_data, body_confidence = body_points(results.pose_landmarks, w, h, 33)
-        face_data, face_confidence = component_points(results.face_landmarks, w, h, 468)
+        face_data, face_confidence = component_points(results.face_landmarks, w, h, 478 if holistic_config['refine_face_landmarks'] else 468)
         lh_data, lh_confidence = component_points(results.left_hand_landmarks, w, h, 21)
         rh_data, rh_confidence = component_points(results.right_hand_landmarks, w, h, 21)
+        # TODO: add pose_world_landmarks
+        # TODO: reduce the face landmarks?
```

Comment:
The most tricky one! From Sarah/Cihan:

> No post processing has been done on the data I have given to you, yes. But we do not save all the key points from the face mesh. We reduce the face landmarks using the following code

<img width="975" alt="image" src="https://user-images.githubusercontent.com/2316987/167448935-71c15ab4-87c9-459f-a4df-b41d5aae9ac1.png">

I do not understand fully what they are doing here based on the screenshot, but anyway they reduce the number of landmarks somehow, how would we implement this?

I think maybe it should not be implemented here, instead we always store everything, let the user on downstream tasks decide how many landmarks to use.

I will need more class clarification with Sarah on this.